apis/fluentbit/v1alpha2/plugins/filter/kubernetes_types.go

```diff
@@ -65,6 +67,26 @@ type Kubernetes struct {
 	KubeMetaPreloadCacheDir string `json:"kubeMetaPreloadCacheDir,omitempty"`
 	// If set, use dummy-meta data (for test/dev purposes)
 	DummyMeta *bool `json:"dummyMeta,omitempty"`
+	// DNS lookup retries N times until the network start working
+	DNSRetries *int32 `json:"dnsRetries,omitempty"`
+	// DNS lookup interval between network status checks
+	DNSWaitTime *int32 `json:"dnsWaitTime,omitempty"`
+	// This is an optional feature flag to get metadata information from kubelet
+	// instead of calling Kube Server API to enhance the log.
+	// This could mitigate the Kube API heavy traffic issue for large cluster.
+	UseKubelet *bool `json:"useKubelet,omitempty"`
+	// kubelet port using for HTTP request, this only works when useKubelet is set to On.
+	KubeletPort *int32 `json:"kubeletPort,omitempty"`
+	// kubelet host using for HTTP request, this only works when Use_Kubelet set to On.
+	KubeletHost string `json:"kubeletHost,omitempty"`
+	// configurable TTL for K8s cached metadata. By default, it is set to 0
+	// which means TTL for cache entries is disabled and cache entries are evicted at random
+	// when capacity is reached. In order to enable this option, you should set the number to a time interval.
+	// For example, set this value to 60 or 60s and cache entries which have been created more than 60s will be evicted.
+	KubeMetaCacheTTL string `json:"kubeMetaCacheTTL,omitempty"`
+	// configurable 'time to live' for the K8s token. By default, it is set to 600 seconds.
+	// After this time, the token is reloaded from Kube_Token_File or the Kube_Token_Command.
+	KubeTokenTTL *int32 `json:"kubeTokenTTL,omitempty"`
```

Comment:
The `Kube_Token_TTL` is a time interval, so it should be `string`.

Reply:
Thanks, fixed